pyth_observer/check/price_feed.py

```diff
@@ -64,28 +65,30 @@ def run(self) -> bool:
         if not is_market_open:
             return True
 
-        # Skip if not trading
-        if self.__state.status != PythPriceStatus.TRADING:
-            return True
-
         distance = abs(
-            self.__state.slot_aggregate_attempted - self.__state.slot_aggregate
+            self.__state.latest_block_slot - self.__state.latest_trading_slot
```

Comment:
A price feed can go stale without its status becoming non-trading. This check relies on checking the latest slot with the latest trading slot (which includes non-trading checks before)